File: hibernate-query-language/src/main/java/io/github/robertomike/hefesto/actions/wheres/WhereRaw.java
package io.github.robertomike.hefesto.actions.wheres;

import io.github.robertomike.hefesto.enums.WhereOperator;

public class WhereRaw extends BaseWhere {
    public String query;

    public WhereRaw(String query) {
        this.query = query;
        this.whereOperation = WhereOperator.AND;
    }

    public WhereRaw(String query, WhereOperator whereOperator) {
        this.query = query;
        this.whereOperation = whereOperator;
    }
}

File: hibernate-query-language/src/main/kotlin/io/github/robertomike/hefesto/actions/wheres/WhereRaw.kt
package io.github.robertomike.hefesto.actions.wheres

import io.github.robertomike.hefesto.enums.WhereOperator

class WhereRaw(
    val query: String,
    whereOperator: WhereOperator = WhereOperator.AND
) : BaseWhere() {
    init {
        this.whereOperation = whereOperator
    }
}
